### ISSUE TITLE
open namespaced schema location fails

### ISSUE THREAD

**Issue by Morgy93** [redhat-developer/vscode-xml#318]
Hello,

I don't know if it's just me or some bug, that's why I'm here. =)

I'm using Magento 2 and schema validation works just fine (so I assume that there is no error here) but if I try to "Follow link" on namespaced xsd it fails:

![image](https://user-images.githubusercontent.com/7961978/91647148-7e492a80-ea57-11ea-9d8a-0387ea68c578.png)

Sample xml:

```xml
<?xml version="1.0"?>
<!--
/**
 * Copyright © Magento, Inc. All rights reserved.
 * See COPYING.txt for license details.
 */
-->
<page xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:View/Layout/etc/page_configuration.xsd">
    <body>

    </body>
</page>

```

catalog.xml
```xml
<?xml version="1.0"?>
<catalog xmlns="urn:oasis:names:tc:entity:xmlns:xml:catalog">
    ...
    <system systemId="urn:magento:framework:View/Layout/etc/page_configuration.xsd" uri="/path/to/magento/vendor/magento/framework/View/Layout/etc/page_configuration.xsd"/>
    ...
</catalog>
```

Any obvious reason why it fails?

**Comment by angelozerr** [comment]
> Any obvious reason why it fails?

It's a bug which is perhaps fixed in 1.4.0 (not released) with  https://github.com/eclipse/lemminx/issues/850

Could you try to install last version of vsix https://download.jboss.org/jbosstools/vscode-xml/staging/?C=M;O=D and tell me if it works? Thanks!

**Comment by Morgy93** [comment]
@angelozerr Yes it works beautifully!
I'll close this issue since it will then be automatically fixed with the new release, thanks!
